packages/reporter/src/main.tsx

```diff
@@ -118,7 +120,7 @@ class Reporter extends Component<ReporterProps> {
 
     if (
       this.props.resetStatsOnSpecChange &&
-      this.props.spec.relative !== newProps.spec.relative
+      this.props.specRunId !== newProps.specRunId
```

Comment:
How come we are using an `id` instead of the path (which is pretty much like an `id` anyway)?

Reply:
Not really. The root of issue itself that when we have same spec, it's a different run anyway.